### ISSUE TITLE
Missing playpen icons on mobile safari / chromium (iPhone 6)

### ISSUE THREAD

**Issue by budziq** [azerupi/mdBook#325]
On an iphone 6 I was able to borrow for a few minutes and on browserstack.com all of playpen buttons are hidden both on mobile safari / chromium standard and private mode). This happens even before my recent changes to local storage.

The buttons are visible only if the theme does not load properly (and then they are at the left instead of right side of playpen).
![browserstack](https://user-images.githubusercontent.com/221000/27027102-6a2d61b4-4f60-11e7-93bf-394437e08e20.png)

@azerupi I admit that I am out of my depth ATM with my 2week webdev experience :smile_cat: most likely something with theaming results in hidden icons fontawesome icons.
Moreover, I do not have access to IOS device. As a crutch browserstack.com can be used (it seams quite usable and gives access to chorme/safari developer console as seen on the screenshot) but I have used-up the free 30min usage limit already.

I have reached out to browserstack staff if they would be willing to provide mdBook project with some free plan for opensource project as they advertise on [their webpage](https://www.browserstack.com/accounts/subscriptions), we'll see what they respond. I hope that you don't mind (I have mentioned that I am just a regular contributor without control on the project policy)

**Comment by azerupi** [comment]
I can't reproduce this in any of the browser simulators (chrome and safari) when emulating an iPhone screen. So I suppose this is a bug specific to the iPhone.

> I have reached out to browserstack staff if they would be willing to provide mdBook project with some free plan for opensource project as they advertise on their webpage

Interesting, let me know what they say 😉

**Comment by budziq** [comment]
> I can't reproduce this in any of the browser simulators (chrome and safari) when emulating an iPhone screen. So I suppose this is a bug specific to the iPhone.

Yeah it seems to be a problem with actual device maybe even on specific firmware. I was able to reproduce it with physical iPhone and with browserstack iPhone 6 connection (this is supposedly a direct connection to a physical device with some remote desktop. It is laggy and choppy enough that I believe it ;) )

> Interesting, let me know what they say :wink:

Well they have responded quickly but unfortunately they have requested to add a badge and blurb :slightly_frowning_face: to project webpage or github repo.
[BrowserStack Ticket # 45306 Open source request.pdf](https://github.com/azerupi/mdBook/files/1067696/BrowserStack.Ticket.45306.Open.source.request.pdf)

From my point of view it might be worth it but I guess that you would have to check out their service and decide :/

**Comment by azerupi** [comment]
>they have requested to add a badge and blurb 🙁 to project webpage or github repo

It's tempting and I understand their request. It's only fair when they offer you free access to a paid product, but I would prefer not to be used as advertising so I think I'm going to respectfully decline the offer.

I might be able to get a hold of an iPhone sometimes to test and for the rest I hope we can rely on occasional contributors / users who would be willing to test some things for us.

**Comment by budziq** [comment]
More than understandable :smiley:  I'll forward your response to them.

**Comment by budziq** [comment]
I was able to reproduce this problem with iPhone 7 simulator (IOS 10.3)
![saucelabs com](https://user-images.githubusercontent.com/221000/27034310-6534e304-4f7d-11e7-92b1-e61216dec5c6.png) on virtual image provided via saucelabs (unfortunately they do not provide a developer console so I will leave this issue to someone else).

On positive side it looks like anyone with access to MacOS willing to install xCode would be able to help with this issue :)

**Comment by azerupi** [comment]
>On positive side it looks like anyone with access to MacOS willing to install xCode would be able to help with this issue

How interesting! I have a MacOS desktop, I will see if I can get that working after my exams.
